spec/support/api/helpers.rb

```diff
@@ -50,6 +50,13 @@ def collection_action_identifier(type, action, method = :post)
           action_identifier(type, action, :collection_actions, method)
         end
 
+        def collection_action_identifiers(type, action, method = :post, klass)
```

Comment:
dup method name as 49, shouldn't this be collection_action_classed_identifier ?

Reply:
also, wasn't that just merged ? Maybe a rebase will pick it up.

Reply:
Erps yeah sorry don't look for a minute